### ISSUE TITLE
UI Font size for Tree menu nodes of type folders doesn't work

### ISSUE THREAD

**Issue by archenroot** [dbeaver/dbeaver#12975]
So I try to enlarge fonts of folders (not files) in boht Database Navigator and Project tabs, but it only seem like file nodes can be enlarged, the folder type nodes stay small as seen on this screen:
![image](https://user-images.githubusercontent.com/553512/123263762-9d8ada00-d4f9-11eb-8f18-e872d58c8273.png)

I enlarged in Global Settings Color Schema all font options to 18, but the menu items still remain small. Let me please know where to enlarge font? Similarly, you can see that top menu font is also small, I don't know where to make it bigger.

#### System information: 
- Operating system (distribution) and version: Gentoo AMD64
- DBeaver version - 21.1.1
- Java version 
- openjdk version "1.8.0_292"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_292-b10)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.292-b10, mixed mode)

**Comment by archenroot** [comment]
Ok, I found this related very old issue: https://github.com/dbeaver/dbeaver/issues/964
But its not solution on dbeaver level...

**Comment by uslss** [comment]
thank you for the report

**Comment by flambchap** [comment]
We also want to apply the `Tree and Table fon for views` to Navigator.
